web/admin/application/configs/klear/model/CallCsvSchedulers.yaml

```diff
@@ -98,36 +98,39 @@ production:
       attributes:
         data-autofilter-select-by-data:
           company: type
-      required: true
+          vpbx: type
+          retail: type
+          residential: type
+      required: false
       source:
         data: inline
         values:
-          '_NULL_':
+          '__NULL__':
             title: _('All')
             visualFilter:
               show: ["callCsvNotificationTemplate"]
-              hide: ["company", "residentialDevice", "retailAccount", "ddi"]
+              hide: ["company", "retail", "residential", "vpbx", "residentialDevice", "retailAccount", "ddi", "user", "fax", "friend", "endpointType"]
           vpbx:
             title: _('vPBX')
             visualFilter:
-              show: ["company"]
-              hide: ["residentialDevice", "retailAccount", "callCsvNotificationTemplate"]
+              show: ["vpbx"]
+              hide: ["retail", "residential", "company", "residentialDevice", "retailAccount", "callCsvNotificationTemplate"]
           retail:
             title: _('Retail')
             visualFilter:
-              show: ["company", "retailAccount"]
-              hide: ["residentialDevice", "callCsvNotificationTemplate"]
+              show: ["retail"]
+              hide: ["company", "vpbx", "residential", "residentialDevice", "callCsvNotificationTemplate", "user", "fax", "friend", "endpointType", "retailAccount"]
           residential:
             title: _('Residential')
             visualFilter:
-              show: ["company", "residentialDevice"]
-              hide: ["retailAccount", "callCsvNotificationTemplate"]
+              show: ["residential"]
+              hide: ["company", "vpbx", "retail", "retailAccount", "callCsvNotificationTemplate", "user", "fax", "friend", "endpointType", "residentialDevice"]
           wholesale:
             title: _('Wholesale')
             visualFilter:
               show: ["company"]
-              hide: ["retailAccount", "residentialDevice", "callCsvNotificationTemplate"]
```

Comment:
This is ok, but if you feel the need to include loots of entries in yaml arrays, it's pereferred to used the multiline list format. Also, not applicable here, but show entries have preference over hide, so a field in both places will be displayed. This is handy in some scenarios like [this one](https://github.com/irontec/ivozprovider/blob/bleeding/web/admin/application/configs/klear/model/Ddis.yaml#L50)